pkgs/servers/sql/percona-server/8_0.nix

```diff
@@ -14,8 +49,16 @@ stdenv.mkDerivation (finalAttrs: {
     hash = "sha256-zZgq3AxCRYdte3dTUJiuMvVGdl9U01s8jxcAqDxZiNM=";
   };
 
-  nativeBuildInputs = [ bison cmake pkg-config ]
-    ++ lib.optionals (!stdenv.isDarwin) [ rpcsvc-proto ];
+  nativeBuildInputs = [
+    bison
+    cmake
+    pkg-config
+    makeWrapper
+    # required for scripts/CMakeLists.txt
+    coreutils
+    gnugrep
+    procps
+  ] ++ lib.optionals (!stdenv.hostPlatform.isDarwin) [ rpcsvc-proto ];
```

Comment:
This is not the diff from before on 24.05, but includes https://github.com/NixOS/nixpkgs/commit/a5dcd41e27c1330eb8333951938511c3a5e97f9c.

I'm fine with including this as well, however please do so as separate commit.
Sorry for the extra work here, but a commit called claiming to reformat files must not have functional changes in it.

Reply:
Now the diff mismatch results from explicitly _not_ backporting the changes from e0464e47880a69896f0fb1810f00e0de469f770a.